pyproject.toml

```diff
@@ -1,8 +1,15 @@
 [project]
+name = "llama-stack-provider-kft"
+version = "0.1.0"
+description = "Llama Stack Provider for Distributed InstructLab Training using KFT"
+readme = "README.md"
+requires-python = ">=3.10"
 dependencies = [
     "llama-stack>=0.2.3",
     "kubeflow-training==1.9.1",
-    "kubernetes",
+    "kubernetes==30.1.0",
+    "click==8.1.8",
+    "instructlab-training==0.7.0",
```

Comment:
Can you create a new extra dependency section for these two as they are not strictly needed for the project to operate. I believe it's only needed for the `utils` directory. Thanks!

Reply:
Done!